src/inverseRobot.js

```diff
@@ -5,9 +5,31 @@
  *
  * @return {object}
 */
-
 function inverseRobot(robot) {
   // write code here
+
+  let newRobot = {};
+  const passedValues = [];
+  const keys = Object.keys(robot);
+  const values = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
+    newRobot[values[i]] = keys[i];
+  }
+
+  for (let i = 0; i < values.length; i++) {
```

Comment:
Do you really need one more loop? Can't you do all the checks in the previous one?
Also, what's the purpose of `passedValues` array? You can search for duplicated values if `newRobot`. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in